src/axom/sidre/spio/IOManager.cpp

```diff
@@ -1086,10 +1086,18 @@ void IOManager::writeBlueprintIndexToRootFile(DataStore* datastore,
   AXOM_DEBUG_VAR(root_file_id);
   SLIC_ASSERT(root_file_id >= 0);
 
-  std::string bp_index("blueprint_index/" + mesh_name);
+  std::string blueprint_name;
+  std::string path_to_mesh;
+  std::string slash = "/";
```

Comment:
[I don't imagine this will change anytime soon, but] 
this function should probably be using ``sidre::Group::s_path_delimiter`` rather than "slash".

Reply:
@kennyweiss the access code is non trivial -- best way to validate is using VisIt to read. 

I guess what you are after is some check for the generic path construction. I wan't to move that to conduit but its not there yet (it's in visit and ascent)?

Reply:
https://github.com/visit-dav/visit/blob/develop/src/databases/Blueprint/avtBlueprintTreeCache.C

Reply:
Thanks @cyrush -- so, should we just wait for this functionality to be available in conduit and then add a call here?

Reply:
I've changed the delimiter--to access it I have to call a method on the root group, even though it's static data.  Should Group have a static accessor method for this char?

Reply:
I noticed that too -- it definitely should.

Right now, it has a member function to access the static variable.  This function should probably just be made static.

https://github.com/LLNL/axom/blob/1043eeed4973a464b395cd6af93c5e2562015238/src/axom/sidre/core/Group.hpp#L131-L137